contents/gpgpu/gpgpu.md

```diff
@@ -48,6 +50,18 @@ The SIMT (Single Instruction, Multiple Threads) architecture of GPUs means that
 - CUDA: [Official website](http://www.nvidia.com/object/cuda_home_new.html)
 - OpenCL: [Official website](https://www.khronos.org/opencl/)
 
+### § 1.6 GPGPU in action
```

Comment:
I almost missed this, but to be consistent, it would be great to update the table of contents at the top of the document too.